README.md

```diff
@@ -2,27 +2,28 @@
+| [github__daily_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__daily_metrics)     | Each record represents a single day, enriched with metrics about PRs and issues that were created and closed that day.                              |
+| [github__weekly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__weekly_metrics)    | Each record represents a single week, enriched with metrics about the PRs and issues that were created and closed that week.                             |
+| [github__monthly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__monthly_metrics)   | Each record represents a single month, enriched with metrics about PRs and issues that were created and closed that month.                            |
+| [github__quarterly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__quarterly_metrics) | Each record represents a single quarter, enriched with metrics about PRs and issues that were created and closed that quarter.                          |
 
-# 🤔 Who is the target user of this dbt package?
-This package is intended to be leveraged by individuals who want to take advantage of analysis ready models to fuel their reporting needs. The package outputs reporting tables that may be easily linked directly to your BI tool of choice for quick time to value with your GitHub data.
+# 🤔 Why should I use this dbt package?
+This package is designed for users who want to use pre-built, analysis-ready models to fuel their reporting needs. The package creates reporting tables that you link directly to your BI tool for quick time to value with your GitHub data.
 
-This package models the data in a way that it will effectively answer the majority of your questions. However, it will never answer **all** of your business questions. That being said, you are always able to reference the models from this package and apply your own additional custom transformations to fit your specific reporting needs. If you believe this package should include additional models and believe other users will benefit, I recommend you refer to the [Contributions]() section for information on how to contribute to the open-source package.
+This package models the data so that it can effectively answer most of your business questions. However, it will never answer _all_ of your questions. If this package doesn't quite fit your reporting needs, you can still reference the models from this package and apply your own additional custom transformations. If you think that this package should include additional models and believe other users will benefit, please refer to the [Contributions]() section to learn how to contribute to the open-source package.
 
 # 🎯 How do I use the dbt package?
-To effectively install this package and leverage the pre-made models, you will want to follow the below steps:
+To install this package and leverage the pre-built models, do the following:
```

Comment:
What do you think about making a Prerequisites section, similar to the ones we have for connectors? https://fivetran.com/docs/applications/github/setup-guide#prerequisites